ui/client/datasets/annotations/dataOUT.js

```diff
@@ -125,6 +125,13 @@ function formatDateAnnotationOUT(localAnnotation, outgoingAnnotationBase) {
     genDateMultiPartMemberAnnotation(localAnnotation, outgoingAnnotation)
       .forEach((item) => { collectedOutgoingAnnotations.push(item); });
   }
+
+  if (localAnnotation.date_type == 'epoch') {
+    // fix for cartwrights/geotimeclassify inserting time_format for epochs, which is not needed
+    // also helps with csv dictionary annotation
```

Comment:
(meaning, helps in the case someone enters invalid, unnecessary, time_format(s) on their csv,xls dictionary annotation file column).